### ISSUE TITLE
Change default username 'user' when creating a standalone VM

### ISSUE THREAD

**Issue by sucrecacao** [QubesOS/qubes-issues#7984]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### The problem you're addressing (if any)

There is a very easy way to get quickly productive when swithing to a new linux computer

 1. create a new user in the new machine with the same username as the old computer
 2.  then delete the $HOME/username of the new computer 
 3.  paste the $HOME/username of the old machine into the $HOME of the new machine

I would like to do the same in qubes, using a standalone VM. But I can't because the default root username is 'user', and I need it to be the same as my old username. 

i was not able to change the 'user' name to another one, neither to create a standalone VM with a different username. The solution I found is to simply add a new sudo user with my old username, which is annoying because but app launcher will start app from 'user', same for file transfer that will go to /home/user/QubesIncomming instead /home/myOldUsername/QubesIncomming.

### The solution you'd like

Be able to select the name of the root user when creating a new standalone VM

### The value to a user, and who that user might be

Every user that want to seamlessly switch to Qubes. 
They just need to copy their old /home to a standalone vm, and just forget about the old computer. 
Then, they can little by little dispatch their file to new App VM while still having a handy 'legacy' standalone vm.

**Comment by jamke** [comment]
I think allowing different user names/UID/GID for default user in qubes is a bad idea.
Just stick to the `user` and everything will work. 

@sucrecacao if you want to migrate your old user profile from GNU/Linux - just use scripts to change ownership (like recursive chown). What is the problem? It's a doable task that had to be made only once. And it does not cause a lot of future problems with situations when there are non-standard or multiple users.

P.S. Not only username can be different but user ID (UID). And files are `chown`-ed based on IDs, not actual names. So, the only reasonable way - to simply `chown` your files of old $HOME to belong to `user` with a default user and group ID (1000:1000). For many distributions this is not even needed, because those have the same UID:GID for default user out of the box.

**Comment by sucrecacao** [comment]
> What is the problem? It's a doable task that had to be done only once. And it does not cause a lot of future problems with situations when there are non-standard or multiple users.

The problem is that a lot of scripts, compilation, and installation will have hardcoded link like "/home/oldusername/.../something".
Therefore it is not possible to stick to the "paramount to keep the old username for a seamless integration of the old sytem into qubes, 

>  Just stick to the user and everything will work.

In the opposite, a lot of thing will break. 

> I think allowing different user names/UID/GID for default user in qubes is a bad idea.

I currently have a standalone VM with an different username/uid/gid and it works well in itself, it just integrate poorly with the rest of qubes' ecosystem.

**Comment by jamke** [comment]
1) How do you transfer you user profile from on one PC with Ubuntu and your user `user1` (UID 1000) to another PC with Ubuntu that already has a `user2` who uses UID 1000? Still use the same scripts to chown to UID 1001 or something?

2) And why do you say `a lot of scripts`? You can recursively `chown` with a single command. Or take some ten-line script to recursively chown with additional checks of ownership.

3)  I think it is not a good idea to add additional complications to the system that `by design` provides single-user qubes without ability to change logins in those Qubes. Complications for no additional improvements, actually.

4) Solution for your scenario: use `user` on GNU/Linux with default Qubes OS's UID/GID and transfer your profile between GNU/Linux and Qubes OS without any additional work. That's so simple, everything will work as you want - like a fast profile migration without `chown`-ing. I had been using `user` as a username on my GNU/Linux devices before Qubes OS was created, no issues so far.

**Comment by sucrecacao** [comment]
@jamke I'm cutting the conversation right here. Please let's not add noise and out-of-scope topic to the conversation and let's focus solving the issue. If you want to continue elsewhere the conversation, feel free to PM me on matrix

**Comment by jamke** [comment]
> The problem is that a lot of scripts, compilation, and installation will have hardcoded link like "/home/oldusername/.../something".

Sorry, I misread the problem you were saying. I though that a lot of scripts rely on ownership, while you were actually talking about paths, right?

I would advice to simply create a relative symlink from `/home/oldusername/` to `/home/user/` with a proper ownership. I did things like that many times when I had problems with hard-coded paths and absolute symlinks that had outdated paths. Works well in almost all cases.

**Comment by ben-grande** [comment]
Yep, it's broken:

```
% qvm-create --standalone -t debian-13-minimal -l red --prop=default_user=alice test-std

% qvm-prefs test-std default_user
alice

% qvm-run -p test-std 'echo $USER $HOME'
2026-01-21 11:36:36.574 qrexec-agent[516]: qrexec-agent.c:189:do_exec: user alice does not exist
zsh: exit 125   qvm-run -p test-std 'echo $USER $HOME'

% qvm-shell test-std
2026-01-21 11:36:57.250 qrexec-agent[536]: qrexec-agent.c:189:do_exec: user alice does not exist
zsh: exit 125   qvm-shell test-std

% qvm-shell -uroot test-std
root@test-std:~# ls /home
user
```

> i was not able to change the 'user' name to another one, neither to create a standalone VM with a different username. The solution I found is to simply add a new sudo user with my old username, which is annoying because but app launcher will start app from 'user', same for file transfer that will go to /home/user/QubesIncomming instead /home/myOldUsername/QubesIncomming.
> ###

Set property `default_user`:
```
qvm-prefs default_user QUBE USER
```
